oracle/.dockerignore

```diff
@@ -1,12 +1,6 @@
-CODE_OF_CONDUCT.md
-CONTRIBUTING.md
-docker-compose.yml
-Dockerfile
-.dockerignore
-docs
-e2e
-.env*
-.git*
-LICENSE
-README.md
-node_modules
+**/node_modules
+**/submodules
+**/Dockerfile
+**/.dockerignore
+**/docker-compose.yml
+.gitmodules
```

Comment:
Do you think the following files and directories are needed to be included in the oracle docker image?
```
docs
test
README.md
*.png
```

Reply:
Let's remove them in #66 , in which I will probably be able to use only one `.dockerfile`

Reply:
I have updated the `.dockerignore` on both PRs.